pkg/scheduler/framework/runtime/framework.go

```diff
@@ -734,6 +735,7 @@ func (f *frameworkImpl) RunFilterPlugins(
 		if state.SkipFilterPlugins.Has(pl.Name()) {
 			continue
 		}
+		metrics.PluginEvaluationTotal.WithLabelValues(pl.Name(), Filter, f.profileName).Inc()
```

Comment:
🤔 why is the Filter value exported, but not preFilter? But this is outside of the scope of this PR.

Reply:
Seems `Filter` value is used in another package. (schedule_one.go)
https://github.com/sanposhiho/kubernetes/blob/799444a689da80470ad1aa87153f4feb5134b3db/pkg/scheduler/schedule_one.go#L533

It may be better to move those variables to metrics package? since they are used in the metric value's label.

Reply:
yes, that sounds like a better location.

Reply:
PTAL https://github.com/kubernetes/kubernetes/pull/116312